release_task.rb

```diff
@@ -63,10 +63,60 @@ def post_filename
     "#{@release_date.strftime("%F")}-#{@product_id}-#{@version}.md"
   end
 
+  def product_install_url
+    "/docs/install.html"
+  end
+
+  def product_release_note_url
+    major_version = @version.split(".")[0];
+    "/docs/news/#{major_version}.html#release-#{@version.gsub(".", "-")}"
+  end
+
+  def post_ja_content
+    <<-CONTENT
+---
+layout: post.ja
+title: #{@product} #{@version}リリース
+description: #{@product} #{@version}をリリースしました！
+---
+
+## #{@product} #{@version}リリース
+
+#{@product} #{@version}をリリースしました！
+
+それぞれの環境毎のインストール方法は、[インストール](/ja#{product_install_url})をご確認ください。
+
+主な変更点のついては、[リリースノート](/ja#{product_release_note_url})をご確認ください。
+    CONTENT
+  end
+
+  def post_en_content
+    <<-CONTENT
+---
+layout: post.en
+title: #{@product} #{@version} has been released
+description: #{@product} #{@version} has been released!
+---
+
+## #{@product} #{@version} has been released
+
+#{@product} #{@version} has been released!
+
+For installation instructions on your environments, please see the [Installation Guide](#{product_install_url}).
+
+For the information on the changes, please see the [Release Note](#{product_release_note_url}).
+    CONTENT
+  end
+
   def post_content(locale)
-    # TODO: We will write blog post contents here.
-    # After writing contents, we will remove this TODO comment.
-    "#{locale}, #{@product}, #{@version}"
+    case locale
+    when "ja"
+      post_ja_content
+    when "en"
+      post_en_content
+    else
+      raise "#{locale} isn't supported for release announce posts in blog."
+    end
```

Comment:
```suggestion
    __send__("post_#{locale}_content")
```

`post_content_#{locale}`にして`post_content`をprefixにして関連処理であることをアピールしたほうがいいかも。

Reply:
fix: [ed63a9e](https://github.com/groonga/groonga.org/pull/87/commits/ed63a9e13a23e181937f56026166684ddc5552fe) 
ありがとうございます。わざわざ例外を挙げなくても、NoMethodErrorで気づけますし何をしているか、たしかにわかりやすいですね 🙏🏾 

> post_content_#{locale}にしてpost_contentをprefixにして関連処理であることをアピールしたほうがいいかも。